src/controllers/controller_impls/session_controller.rs

```diff
@@ -112,6 +123,12 @@ impl SessionControllerTrait for SessionController {
                     TokenType::RefreshToken,
                     request
                         .token_str()
+                        .map_err(|err| {
+                            Status::internal(format!(
+                                "failed to convert token to string, internal error: {}",
+                                err
+                            ))
```

Comment:
Her returneres error beskeden, mens den ikke gøres det de andre steder vi har lavet en `map_err` på `token_str`. Måske de andre også skal gøre det på denne måde, eller ændre denne alt efter om vi gerne vil sende den interne error med tilbage eller ik.

Reply:
så vidt jeg ved bliver kald af `token_str`/`token_string``.map_err(...)` altid efterfulgt at `?` for at returnere tidligt (undtagen i tests)

Reply:
Det også rigtigt, men min pointe var mere, at error beskeden i dette eksempel er formateret og inkluderer `err` til slut, mens det ikke gøres de andre steder vi kalder `token_string`.

Reply:
nårh, fair nok. kan godt inkludere indre fejlbeskeder i andre kaldsteder